.buildscript/update_docs.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+echo "Cloning osstrich..."
+mkdir tmp
+cd tmp
+git clone git@github.com:square/osstrich.git
```

Comment:
- This is going to always clone master which means builds can change in behavior. You may want to ping this
- Consider cloning this inside `~/.gradle/caches` since travis will cache that dir and avoid unnecessary fresh clones on every build everytime

Reply:
This isn't something run on CI or otherwise automated

Reply:
oh gotcha. sry I thought i saw it on travis. nvm :)

Reply:
:)